crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -245,8 +245,15 @@ pub fn main(cfg: &mut Config) -> Result<()> {
     println!("Saving sandbox config");
     cfg.save()?;
 
+    // Run node
+    let jstz_node = OctezThread::from_child(
+        Command::new("cargo")
+            .args(["run", "--bin", "jstz-node"])
+            .spawn()?,
+    );
+
```

Comment:
We shouldn't be using `cargo` like this, since this won't be packaged correctly on the user's installation (only works in our local dev environment)

Reply:
yes, we need to compile it and then change to just 'jstz-node'. For now however, I think it can stay like this and we can create a separate task for changing it.

Reply:
I'd be keen on tracking it in this PR since the other changes in this PR would be mostly unnecessary if this was our desired approach